src/clockkit.cs

```diff
@@ -77,6 +110,14 @@ interface CLKComplicationDataSource {
 		[Watch (3,0)]
 		[Export ("getLocalizableSampleTemplateForComplication:withHandler:")]
 		void GetLocalizableSampleTemplate (CLKComplication complication, Action<CLKComplicationTemplate> handler);
+
+		[Watch (7, 0), NoiOS]
+		[Export ("getComplicationDescriptorsWithHandler:")]
+		void GetComplicationDescriptors (Action<NSArray<CLKComplicationDescriptor>> handler);
```

Comment:
custom delegate ?

other API use `CLKComplicationDescriptor[]` not an `NSArray` so that might face conversions

Reply:
Do we need the custom delegate? is a single arg with the complications supported by the app.